core/orchestrator.go

```diff
@@ -198,6 +198,8 @@ func (orch *orchestrator) ProcessPayment(ctx context.Context, payment net.Paymen
 			ev := ticket.EV()
 			orch.node.Balances.Credit(sender, manifestID, ev)
 			totalEV.Add(totalEV, ev)
+			totalFaceValue.Add(totalFaceValue, ticket.FaceValue)
+			totalWinProb.Add(totalWinProb, ticket.WinProbRat())
```

Comment:
1. So, we never set those to `0` again and the log will always show the cumulative tickets?
2. I wonder how much sense it takes to add the probabilities and face values together? What this information actually may give you?

Reply:
1. These variables are initialized in the ProcessPayment function so start at 0 every time. The log line is intended to show the cumulative amounts for the tickets (face value/win prob/ev).   
2. To me, this is a debug log line so more info is likely better right?

Reply:
Ahh, you're right. I missed that part that they are initialized for each payment processing. Thansk for the explanation!